src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1834,11 +1834,14 @@ public async Task PostAsync_CallMethod_EmptyContent(Uri remoteServer)
         [InlineData(false, "1.1")]
         [InlineData(true, "1.1")]
         [InlineData(null, "1.1")]
+        [InlineData(false, "2.0")]
+        [InlineData(true, "2.0")]
+        [InlineData(null, "2.0")]
         public async Task PostAsync_ExpectContinue_Success(bool? expectContinue, string version)
```

Comment:
Is this the only test we have for Expect: 100-Continue? Seems like we should at least have a test for the failure case too.

Also, what about other intermediate response codes? I thought we added tests for these when Caesar made all the 1xx changes a while back.

Reply:
no, it is not only one test. But some were not easily generalizable. So I added H2 specific one.